near-sdk-macros/src/core_impl/utils/mod.rs

```diff
@@ -73,3 +73,24 @@ pub(crate) fn extract_vec_type(ty: &Type) -> Option<&Type> {
         _ => None,
     }
 }
+
+pub(crate) fn get_event_args(var: &syn::Variant) -> (Option<LitStr>, Option<LitStr>) {
+    let mut v: (Option<LitStr>, Option<LitStr>) = (None, None);
+    var.attrs.iter().for_each(|attr| {
+        let test = attr.parse_args_with(
+            syn::punctuated::Punctuated::<syn::MetaNameValue, syn::Token![,]>::parse_terminated,
+        );
+        if let Ok(item) = test {
+            item.iter().for_each(|kwargs| {
+                if let syn::MetaNameValue { path, lit: syn::Lit::Str(value), .. } = kwargs {
+                    if path.is_ident("standard") {
+                        v.0 = Some(value.clone());
+                    } else if path.is_ident("version") {
+                        v.1 = Some(value.clone());
+                    }
+                };
+            })
+        }
+    });
+    v
```

Comment:
nit: any reason to pre-wrap `v` as a pair? I personally find that this looks more readable:
```suggestion
pub(crate) fn get_event_args(var: &syn::Variant) -> (Option<LitStr>, Option<LitStr>) {
    let mut standard = None;
    let mut version = None;
    var.attrs.iter().for_each(|attr| {
        let test = attr.parse_args_with(
            syn::punctuated::Punctuated::<syn::MetaNameValue, syn::Token![,]>::parse_terminated,
        );
        if let Ok(item) = test {
            item.iter().for_each(|kwargs| {
                if let syn::MetaNameValue { path, lit: syn::Lit::Str(value), .. } = kwargs {
                    if path.is_ident("standard") {
                        standard = Some(value.clone());
                    } else if path.is_ident("version") {
                        version = Some(value.clone());
                    }
                };
            })
        }
    });
    (standard, version)
```